Control/Monad/Writer/CPS.hs

```diff
@@ -36,23 +36,16 @@ module Control.Monad.Writer.CPS (
     WriterT,
     execWriterT,
     mapWriterT,
-    module Control.Monad,
-    module Control.Monad.Fix,
     module Control.Monad.Trans,
-    module Data.Monoid,
   ) where
 
 import Control.Monad.Writer.Class
 
-import Control.Monad.Trans
+import Control.Monad.Trans (MonadTrans (lift))
```

Comment:
`Control.Monad.Trans` used to re-export `MonadIO(liftIO)` too. I don't think that is a good idea. `MonadTrans(lift)` is somewhat justified, but `MonadIO` seems to be there out-of-blue.